File: LuxeBalloonDecor_website.Server/Models/PersonDetails.cs
using System.ComponentModel.DataAnnotations;

namespace LuxeBalloonDecor_website.Server.Models
{
    public class PersonDetails
    {
        [Key]
        public int personID { get; set; }
        public string firstName { get; set; }
        public string lastName { get; set; }
        public DateTime birthDate { get; set; }
        public int role { get; set; } // Stored as Int compare with enum when getting
        public List<string> testimonials { get; set; }
        public List<ContactInfo> contactDetails { get; set; }
    }
}

File: luxeballoondecor_website.client/src/app/models/PersonDetails.ts
import { ContactInfo } from "./ContactInfo";
import { RoleEnum } from "./Enumerators";

export interface PersonDetails{
  personID: number,
  firstName: string,
  lastName: string,
  birthDate: Date,
  role: RoleEnum,
  testimonials: string[],
  contactDetails: ContactInfo[]
}
